app/lib/import_export.rb

```diff
@@ -323,6 +323,8 @@ module ImportExport # rubocop:todo Metrics/ModuleLength
     primary_telephone: 'Primary Telephone',
     email: 'Email',
     contact_attempts: 'Contact Attempts',
+    last_date_of_exposure: 'Last Date of Exposure',
```

Comment:
also need to add these fields to `CLOSE_CONTACTS_EXPORT_OPTIONS` on line 478

Reply:
Thanks for the good feedback. Fixed with latest commit.